spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -268,17 +650,349 @@ public PublisherBuilder<IncomingMessage, OutgoingMessage> processMessages() {
 
 ==== Message acknowledgement
 
+Acknowledgement is an important part of message processing.
 Messages are either acknowledged explicitly, or implicitly by the implementation.
 
-===== Explicit acknowledgement
+Acknowledgement is controlled by the `org.eclipse.microprofile.reactive.messaging.Acknowledgment` annotation. 
```

Comment:
Acknowledgement for `@Incoming` messages is controlled by the

Reply:
(I am not sure it is worth listing all the method shapes below that do not have an _@Outgoing_ annotation.

Reply:
I've removed the `@Outgoing` entries.